base_classes/NXdetector.nxdl.xml

```diff
@@ -758,10 +758,15 @@
     <doc>
       How many bits the electronics reads per pixel.
       With CCD's and single photon counting detectors,
-      this must not align with traditional integer sizes.
+      this might not align with traditional integer sizes.
       This can be 4, 8, 12, 14, 16, ...
     </doc>
   </field>
+  <field name="bit_depth_image" type="NX_POSINT" recommended="true">
+    <doc>
+      The number of bits per pixel saved to the image data.
```

Comment:
The data may be saved as the next higher standard int type (e.g. a 12bit ADC being saved as 16bit uints).